### PR TITLE
[doc] a-text.md, raycasting

### PR DESCRIPTION
**Pull request by andreasplesch**
**Description:**
address raycasting
**Changes proposed:**
- add a limitations sections with a short explanation.

### CODE REVIEW

**Comment by dmarcos** [comment]
We have a cursor example (https://github.com/aframevr/aframe/blob/master/examples/test/cursor/index.html). Do we add some text there to illustrate how to do it?

**Comment by machenmusik** [comment]
I think the `a-plane` labeled with text in https://github.com/aframevr/aframe/blob/master/examples/test/text/scenarios.html would work with cursor, right?  If so, maybe we can point at something like that as example.

**Comment by machenmusik** [comment]
Yeah, IMO the core text component should be thought of more as a specific type of material / decoration and so to be interacted with, the entity needs a geometry (e.g. plane or box) that defines its area / volume for interaction...?

**Comment by andreasplesch** [comment]
A simplistic example on the page may suffice ?

```html
<a-text value="Hello, World!"></a-text>

<!-- for cursor detection add geometry -->

<a-text value="click me!" geometry="primitive:plane"></a-text>

```

**Comment by andreasplesch** [comment]
The wording is friendlier now although I think it is preferable to say explicitly that text itself does not work with cursor or raycaster to avoid any confusion.

File: docs/primitives/a-text.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ Wraps the [text component][text].
 
 ```html
 <a-text value="Hello, World!"></a-text>
+
+<!-- for interactivity add geometry -->
```
Comment (ngokevin) [inline docs/primitives/a-text.md]: I'd add this example below to the limitations, and then:

```<a-text value="Now interactable" geometry="primitive: plane"</a-text>```

Comment (andreasplesch) [inline docs/primitives/a-text.md]: Good idea. Not sure if interactable is a proper word but it is better than interactive.

**Pull request merged**